airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -352,11 +353,13 @@ public void create(final Map<String, String> allLabels,
         .withContainers(mainContainer)
         .withInitContainers(initContainer)
         .withVolumes(volumes)
+        .withNodeSelector(nodeSelectors)
```

Comment:
I think we should include tolerations as well like in `KubePodProcess` https://github.com/airbytehq/airbyte/blob/ebbe5a9a18a59973531d81d3edaf62db734ed323/airbyte-commons-worker/src/main/java/io/airbyte/workers/process/KubePodProcess.java#L510

However, our current node pools don't actually use taints, so we don't need tolerations for this to work for our own cluster.  In general though, we should include it (maybe in a separate followup PR. `buildPodTolerations` looks like a private method though it should be easy to extract out of `KubePodProcess`.)